apps/dashboard/app/(app)/settings/general/update-workspace-name.tsx

```diff
@@ -84,7 +92,7 @@ export const UpdateWorkspaceName: React.FC<Props> = ({ workspace }) => {
             <Button
               variant={updateName.isLoading ? "disabled" : "primary"}
               type="submit"
-              disabled={updateName.isLoading}
+              disabled={updateName.isLoading || isDisabled}
```

Comment:
put it all in here or all in a separate variable, but please don't do it half and half